packages/default-nav/ml/default_navigation.ts

```diff
@@ -109,12 +114,21 @@ export const defaultNavigation: MlNodeDefinition = {
             defaultMessage: 'Data view',
           }),
           link: 'ml:indexDataVisualizer',
+          getIsActive: ({ pathNameSerialized, prepend }) => {
+            return (
+              pathNameSerialized.includes(prepend('datavisualizer')) ||
```

Comment:
nit: for consistency shouldn't we have all path starting with `"/"`? (`prepend('/datavisualizer')`). Same below